feature/experimental/isis/otg_tests/base_adjacencies_test/base_adjacencies_test.go

```diff
@@ -449,58 +454,103 @@ func TestTraffic(t *testing.T) {
 		// disable global hello padding on the DUT
 		global := isis.GetOrCreateGlobal()
 		global.HelloPadding = oc.Isis_HelloPaddingType_DISABLE
-	}, func(isis *ixnet.ISIS) {
-		// disable global hello padding on the ATE
-		isis.WithHelloPaddingEnabled(false)
 	})
+	ts.ATEIntf1.Isis().Advanced().SetEnableHelloPadding(false)
 
-	ate := ts.ATE
+	// ate := ts.ATE
 	// We generate traffic entering along port2 and destined for port1
 	srcIntf := ts.MustATEInterface(t, "port2")
 	dstIntf := ts.MustATEInterface(t, "port1")
+
+	srcIpv4 := srcIntf.Ethernets().Items()[0].Ipv4Addresses().Items()[0]
 	// net is a simulated network containing the addresses specified by targetNetwork
-	net := dstIntf.AddNetwork("net")
-	net.IPv4().WithAddress(targetNetwork.IPv4CIDR()).WithCount(1)
-	net.IPv6().WithAddress(targetNetwork.IPv6CIDR()).WithCount(1)
-	net.ISIS().WithIPReachabilityExternal().WithIPReachabilityMetric(10)
+	netv4 := dstIntf.Isis().V4Routes().Add().SetName("netv4").SetLinkMetric(10)
+	netv4.Addresses().Add().SetAddress(targetNetwork.IPv4)
```

Comment:
Also add the prefix configuration here (even though that's the default, we should derive it from the targetNetwork struct to make the intent clear).

Reply:
The prefix configuration derived from the targetNetwork struct is done here ```netv4.Addresses().Add().SetAddress(targetNetwork.IPv4)``` for IPv4 and here ```netv6.Addresses().Add().SetAddress(targetNetwork.IPv6)``` for IPv6. Could you please clarify?

Reply:
Ah, I mean that the prefix itself should be set explicitly:
netv4.Addresses().Add().SetAddress(targetNetwork.IPv4).SetPrefix(targetNetwork.IPv4Len)

Reply:
Added .SetPrefix(targetNetwork.IPv4Len) and .SetPrefix(int32(targetNetwork.IPv6Len)